src/node_data.cpp

```diff
@@ -13,7 +13,10 @@
 namespace YAML {
 namespace detail {
 
-std::string node_data::empty_scalar;
+const std::string& node_data::empty_scalar() {
+    static const std::string svalue;
+    return svalue ;
```

Comment:
extra space before the semicolon?

clang-format should help you out here.

Reply:
ok, space removed.  
Can we add clang-format to the tests?

Reply:
Thanks. How would we do that?